### PR TITLE
Chore: Add Khroma link to colors subcategory

### LINKED ISSUE #850: [Add] Khroma
### Site URL

https://www.khroma.co/

### Description of the above provided site

Khroma is an innovative color palette generator that simplifies the process of choosing harmonious colors for your design projects.

### Category

Frontend

### Sub Category

colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Oyebolade**
## Fixes Issue
Closes #850

## Changes proposed
I added Khroma link to colors subcategory

## Screenshots
Khroma

### CODE REVIEW

File: database/frontend/colors.json [hunk 1]
```diff
@@ -103,5 +103,12 @@
       "url": "https://www.happyhues.co/",
     "category": "frontend",
     "subcategory": "colors"
+  },
+   {
+    "name": "Khroma",
+    "description": "This is an innovative color palette generator that simplifies the process of choosing harmonious colors for your design projects.",
+      "url": "https://www.khroma.co/",
```
Comment (rupali-codes) [inline database/frontend/colors.json:110]:
```suggestion
    "url": "https://www.khroma.co/",
```

Comment (Oyebolade) [reply database/frontend/colors.json:110]: All right

**Review by CBID2 — APPROVED** [review]
Hi @Oyebolade! :) I applied @rupali-codes's suggestion for you. LGTM! :)

**Review by rupali-codes — APPROVED** [review]

**Comment by Oyebolade** [comment]
Thank you @CBID2 🙏

**Pull request merged**